nym-vpn-android/nym-vpn-client/src/main/scripts/build-libs.sh

```diff
@@ -25,6 +25,7 @@ case  "$(uname -s)" in
 esac
 
 (cd $PWD/../../nym-vpn-core; cargo run --bin uniffi-bindgen generate --library ./target/aarch64-linux-android/release/libnym_vpn_lib.so  --language kotlin --out-dir ../nym-vpn-android/nym-vpn-client/src/main/java/net/nymtech/vpn -n)
+(cd $PWD/../../nym-vpn-core/nym-vpn-lib; cargo license -j --avoid-dev-deps --avoid-build-deps > ./generated/licenses_rust.json)
```

Comment:
I'm  thinking you want to also add `--filter-platform`

Reply:
Also, imho it's simpler to use `--current-dir ..` and then you don't need to move the output file